### ISSUE TITLE
[QA] Fix focus highlights throughout the application

### ISSUE THREAD

**Issue by b1ink0** [rtCamp/next-pms#1662]
## Description

A lot of places in the application are missing our styled focus highlight for input fields and buttons when using tab navigation.

The `frappe-ui-react` components also need to be updated to include these focus highlights, as they are currently missing from some components.

**Comment by b1ink0** [comment]
For QA Team:

Verify that the browser's default blue focus highlight does not appear when using "Tab" to navigate through the application. All keyboard-focusable interactive elements should now use the gray focus highlight instead.

**Comment by renishsurani** [comment]
Hi @PraveenKum11, I found a problem with Firefox where the default blue-highlighted field was visible when I was switching between tabs. We're adding a few reference screenshots because we already have so many.

<img width="1628" height="1280" alt="Image" src="https://github.com/user-attachments/assets/d91229b5-acb8-4fd0-8aa0-922976cdad27" />

<img width="1408" height="1240" alt="Image" src="https://github.com/user-attachments/assets/7e8a310c-8417-4001-86c0-cbbae859f7f7" />

<img width="2524" height="1066" alt="Image" src="https://github.com/user-attachments/assets/c7497068-3689-4d21-9a7d-206f5156db8e" />

<img width="2390" height="986" alt="Image" src="https://github.com/user-attachments/assets/56e2e72e-e696-4f6a-bbe7-a6af9b507c5a" />

<img width="2300" height="1418" alt="Image" src="https://github.com/user-attachments/assets/980aef0f-88b6-4d30-be54-0a95d38cb36e" />

<img width="2512" height="1408" alt="Image" src="https://github.com/user-attachments/assets/1215d8b7-70ca-4f15-82ad-2db85619774a" />

cc: @Ajay-Jaiswal416 @b1ink0

**Comment by renishsurani** [comment]
I have verified that the mentioned issues on Firefox have been resolved.

cc: @Ajay-Jaiswal416

**Comment by renishsurani** [comment]
Verified on the production as an employee.

@Ajay-Jaiswal416, can you please check on the PM side once?

**Comment by Ajay-Jaiswal416** [comment]
Deployment has been completed. I verified the issue in production, and it appears to be resolved. Therefore, I am closing this ticket.
